src/helpers/sendChatMessage.ts

```diff
@@ -59,6 +60,19 @@ export async function sendChatMessage(
     messageBlocks.push(codeBlock);
   }
 
+  // if this is the first time a message is sent, then update the chat title
+  if (chat.title == "Empty Chat") {
```

Comment:
I hesitate to have this condition be based on the default string (set elsewhere, may get changed without realizing it would effect this comparison), but it'd probably work to use the existing check on line 41 that determines if a new chat needs to be made (if that's the same condition, or I might be missing something?)

Reply:
I agree the condition needs to be changed. I changed it so if the set of messages is empty, then the title and tags are set.

Using line 41 as a condition doesn't quite work because some of the time the chat has already been started and initialized prior to sendChatMessage being called.

Reply:
Ah good point good point!